node/tests/Auth.test.ts

```diff
@@ -0,0 +1,305 @@
+import {
+    GlideClient,
+    GlideClusterClient,
+    ProtocolVersion,
+    RequestError,
+} from "..";
+import ValkeyCluster from "../../utils/TestUtils";
+import { getServerVersion, parseEndpoints } from "./TestUtilities";
+
+type TGlideClient = GlideClient | GlideClusterClient;
+
+const NEW_PASSWORD = "new_password";
+const WRONG_PASSWORD = "wrong_password";
+
+describe("Auth", () => {
```

Comment:
In node, we use a bit of different structure than in python, see `SharedTests.ts` for reference.